src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -157,15 +157,18 @@ class SourceTreeItem extends Component<Props, State> {
   }
 
   renderItemName() {
-    const { item } = this.props;
+    const { item, depth } = this.props;
+
+    const name =
+      depth == 0 ? item.name.substr(item.name.indexOf(":") + 1) : item.name;
```

Comment:
what is this doing?

Reply:
This is unmangling the addition of the thread to the URL information which was made in src/utils/sources-tree/addToTree.js